crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -397,11 +418,11 @@ impl Entities {
     }
 
     /// Returns true if the [`Entities`] contains [`entity`](Entity).
-    // This will return false for entities which have been freed, even if
-    // not reallocated since the generation is incremented in `free`
+    /// This will return false for entities which have been freed, even if
```

Comment:
I chose to make these not doc comments, since this is explaining to the reader of the code 'how this does what the doc says'.

That is, this is describing an implementation

Reply:
My bad!